client/src/components/FormAction/README.md

```diff
@@ -15,7 +15,7 @@ Used for form actions. For example a submit button.
    * 'outline-secondary'
    * 'link'
    * 'danger'
- * `icon` (string): The icon to be used on the button, adds `font-icon-{icon}` class to the button. See available icons [here](../../../../fonts/incon-reference.html).
+ * `icon` (string): The icon to be used on the button, adds `font-icon-{icon}` class to a child element in the button. See available icons [here](../../../../fonts/incon-reference.html).
```

Comment:
```suggestion
 * `icon` (string): The icon to be used on the button, adds `font-icon-{icon}` class to a child element in the button. See available icons [here](../../../../fonts/icon-reference.html).
```

Reply:
lol there was a lot wrong with that link. Have fixed it.